cmd/server/cadence/cadence.go

```diff
@@ -174,46 +174,46 @@ func BuildCLI(releaseVersion string, gitRevision string) *cli.App {
 	app.Version = version
 
 	app.Flags = []cli.Flag{
-		cli.StringFlag{
-			Name:   "root, r",
-			Value:  ".",
-			Usage:  "root directory of execution environment",
-			EnvVar: config.EnvKeyRoot,
+		&cli.StringFlag{
+			Name:    "root, r",
```

Comment:
does this need to change to an alias too?

Reply:
yea, I think they do need to change:
```
❯ go run ./cmd/server start -s frontend
Incorrect Usage: flag provided but not defined: -s

[help output]
```
not sure why that isn't an error at startup (e.g. bad cli setup), but it seems to just be silently ignoring the `, r` and similar aliases

Reply:
good catch!